### PR TITLE
Fix headers test

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #438.

### CODE REVIEW

**Comment by Susurrus** [comment]
@GuillaumeGomez These failed the tests on nightly, though I'm not certain why they failed like they did.

**Comment by GuillaumeGomez** [comment]
Because these macros are only used by some features. I'll need to find which and turn them off for others.

**Comment by Susurrus** [comment]
Maybe just allow unused macros?

**Comment by Susurrus** [comment]
`#![allow(unused_macros)]` at the beginning of `src/macros.rs`?

**Comment by GuillaumeGomez** [comment]
I don't like when things are too simple.

**Comment by Susurrus** [comment]
I think you have to put that after the `deny(unused)`.

**Comment by GuillaumeGomez** [comment]
Working!

**Comment by Susurrus** [comment]
LGTM 👍

File: src/lib.rs [hunk 1]
```diff
@@ -7,6 +7,7 @@
 #![cfg(windows)]
 #![allow(bad_style)]
```
Comment (retep998) [inline src/lib.rs]: Why couldn't you have added the `allow(unused_macros)` to this existing line?

Comment (GuillaumeGomez) [inline src/lib.rs]: Seems like it has to be declared after the deny for some reasons...

Comment (retep998) [inline src/lib.rs]: So then why not move it?

**Pull request merged**